applications/RomApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -56,6 +56,18 @@ void  AddCustomStrategiesToPython(pybind11::module& m)
         .def(py::init< LinearSolverType::Pointer, Parameters>() )
         ;
 
+    typedef LSPGROMBuilderAndSolver<SparseSpaceType, LocalSpaceType, LinearSolverType> LSPGROMBuilderAndSolverType;
+
+     py::class_<LSPGROMBuilderAndSolverType, typename LSPGROMBuilderAndSolverType::Pointer, ROMBuilderAndSolverType, BuilderAndSolverType>(m, "LSPGROMBuilderAndSolver")
```

Comment:
Though it is more verbose I'd avoid acronyms when naming these classes, so I'd name `LSPGROMBuilderAndSolver` as `LeastSquaresPetrovGalerkinROMBuilderAndSolver`. Note that this is also consistent with the non least squares Petrov-Galerkin ROM B&S name.